### PR TITLE
[Bluetooth][TCSACR-301] Update Bluetooth guide

### PR DESCRIPTION
**Pull request by Wootak**
Update Bluetooth guide

Signed-off-by: Wootak Jung <wootak.jung@samsung.com>

### CODE REVIEW

File: docs/application/dotnet/guides/connectivity/bluetooth.md [hunk 1]
```diff
@@ -563,15 +563,15 @@ To share data between devices after establishing a connection:
...
+    The received data is passed to the event handler as an instance of the [Tizen.Network.Bluetooth.SocketData](https://samsung.github.io/TizenFX/latest/api/Tizen.Network.Bluetooth.SocketData.html) class, which contains the socket file descriptor, the size of the received data in bytes, and the data itself as a byte[].
 
     ```
     public static IBluetoothServerSocket Socket = null;
 
     public static void DataReceivedServerEventHandler(object sender, SocketDataReceivedEventArgs args)
     {
         BluetoothSetup.Data = args.Data;
-        LogUtils.Write(LogUtils.DEBUG, LogUtils.TAG, "DataReceived in client: " + args.Data.Data);
+        LogUtils.Write(LogUtils.DEBUG, LogUtils.TAG, "DataReceived in client: " + string.Format("0X{0:X2} ", BluetoothSetup.Data.ByteData[0]) + string.Format("0X{0:X2} ", BluetoothSetup.Data.ByteData[1]) + string.Format("0X{0:X2} ", BluetoothSetup.Data.ByteData[2]));args.Data.Data);
```
Comment (mijin85cho) [inline docs/application/dotnet/guides/connectivity/bluetooth.md]: Would you please check `args.Data.Data);` at the end in this line?

Comment (Wootak) [reply docs/application/dotnet/guides/connectivity/bluetooth.md]: Thank you. done.

**Comment by journey2w** [comment]
Please push again for signing....

Command:
```
git commit -a  --amend -s
```

**Comment by Wootak** [comment]
Done.

**Pull request merged**